app/views/shared/_navbar.html.erb

```diff
@@ -21,8 +21,8 @@
           <li class="nav-item dropdown">
             <div class="dropdown-toggle d-flex align-items-center" id="navbarDropdown" data-bs-toggle="dropdown" aria-haspopup="true" aria-expanded="false"><i class="fa-solid fa-circle-user fa-2xl"></i></div>
             <div class="dropdown-menu dropdown-menu-end" aria-labelledby="navbarDropdown">
+              <%= link_to "My reservations", reservation_path(current_user), class: "dropdown-item" %>
```

Comment:
In fixing the conflict between mine and Ricky's branch I made it so that you see your reservations first as I think that makes more sense then seeing your own cards first since we're more of a booking system. that said, can easily just reverse that change and make it so My Cards is first.